Sources/Core/OAFavoritesMapLayerProvider.mm

```diff
@@ -49,27 +49,28 @@
     return _favorites.size();
 }
 
-std::shared_ptr<SkBitmap> OAFavoritesMapLayerProvider::getImageBitmap(const int index)
+std::shared_ptr<SkBitmap> OAFavoritesMapLayerProvider::getImageBitmap(const int index, bool isFullSize = true)
```

Comment:
You only need to provide the default value in the header file and wrap ```/*= true*/``` in the comment.